website/content/api-docs/system/pprof.mdx

```diff
@@ -26,6 +26,44 @@ $ curl \
     http://127.0.0.1:8200/v1/sys/pprof/
 ```
 
+## Allocs
+
+This endpoint returns a sampling of historical memory allocations over the life
+of the program.
+
+| Method | Path              |
+| :----- | :---------------- |
+| `GET`  | `/sys/pprof/allocs` |
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    http://127.0.0.1:8200/v1/sys/pprof/allocs
+```
+
+## Block
+
+This endpoint returns a sampling of goroutines involved in blocking on
+synchronization primitives.
+
+It is included for completeness, but since Vault doesn't normally enable
```

Comment:
Will Vault allow collection of this data in any circumstance (if not normally)?

Reply:
For now only in custom builds, but I intend to add config settings to configure profiling rate.

Reply:
I see. Sounds good. You may want to mention its usefulness in the current scenario and the possibility that you envision, to avoid people thinking that this is useless.

Reply:
Well, it is useless for our primary docs audience atm.  And we don't usually talk about future plans in our docs.